tests/data/snapshots/tests/_support/_generated/DataTesterActions.php

```diff
@@ -941,6 +941,37 @@ public function haveInDatabase($table, $data) {
     public function seeInDatabase($table, $criteria = null) {
         return $this->getScenario()->runStep(new \Codeception\Step\Assertion('seeInDatabase', func_get_args()));
     }
+    /**
```

Comment:
Please delete generated files and .gitignore rule to make sure that they don't get added again.